Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM registry.fedoraproject.org/fedora-minimal:36 AS build
```

Comment:
I think it would be a good idea to have a file comment describing this image and it's intentions, as well as perhaps having the license info.